src/lib/rest/HttpStatusCode.cpp

```diff
@@ -36,6 +36,7 @@ std::string httpStatusCodeString(HttpStatusCode code)
   case SccContextElementNotFound:            return "No context element found"; // Standard HTTP for 404: "Not Found"
   case SccBadVerb:                           return "Method Not Allowed";
   case SccNotAcceptable:                     return "Not Acceptable";
+  case SccConflict:                          return "TooManyResults";
```

Comment:
By the moment (see http://telefonicaid.github.io/fiware-orion/api/v2/), Conflict (HTTP code 409) corresponds only to "TooManyResults" error text. In the future, it may correspond with several error texts (as 400 may correspond to ParseError and BadRquest) and this part may break. However, lest's wait until that situation happens in order to fix.